packages/sdk/tasks/deposit-erc20.ts

```diff
@@ -247,18 +248,35 @@ task('deposit-erc20', 'Deposits WETH9 onto L2.')
     await depositTx.wait()
     console.log(`ERC20 deposited - ${depositTx.hash}`)
 
-    const messageReceipt = await messenger.waitForMessageReceipt(depositTx)
-    if (messageReceipt.receiptStatus !== 1) {
-      throw new Error('deposit failed')
-    }
+    // Deposit might get reorged, wait 10s and also log for reorgs.
+    let prevBlockNumber = 0
+    for (let i = 0; i < 10; i++) {
+      const messageReceipt = await messenger.waitForMessageReceipt(depositTx)
+      if (messageReceipt.receiptStatus !== 1) {
+        throw new Error('deposit failed')
+      }
+
+      if (
```

Comment:
Logging here just so we know a reorg happened

Reply:
TBH I'm kinda surprised that just a block number difference would show up with reorgs, I'd expect to need a block hash check to catch every reorg.